modules/installing-troubleshooting-assisted-installer-oci.adoc

```diff
@@ -4,24 +4,24 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="installing-troubleshooting-assisted-installer-oci_{context}"]
-= Troubleshooting the installation of a cluster on {oci}
+= Troubleshooting the installation of a cluster on {oci-distributed-no-rt}
 
-If you experience issues with using the {ai-full} to install an {product-title} cluster on {oci-first}, read the following sections to troubleshoot common problems.
+If you experience issues with using the {ai-full} to install an {product-title} cluster on {oci-distributed}, read the following sections to troubleshoot common problems.
 
+[id="installing-troubleshooting-load-balancer_{context}"]
+== The Ingress Load Balancer in {oci-distributed-no-rt} is not at a healthy status
 
-== The Ingress Load Balancer in {oci} is not at a healthy status
+This issue is classed as a `Warning` because by using {oci-distributed-no-rt} to create a stack, you created a pool of compute nodes, 3 by default, that are automatically added as backend listeners for the Ingress Load Balancer. By default, the {product-title} deploys 2 router pods, which are based on the default values from the {product-title} manifest files. The `Warning` is expected because a mismatch exists with the number of router pods available, two, to run on the three compute nodes.
```

Comment:
This is a nit, but maybe in a future update, use a consistent style to refer to numbers: either use numerals (2) or words (two) consistently.